package.json

```diff
@@ -122,18 +131,21 @@
   "keywords": [
     "react",
     "redux",
-    "starter",
-    "seed"
+    "redux-saga",
+    "reselect"
   ],
   "repository": {
     "type": "git",
-    "url": "git://github.com/rangle/react-redux-starter.git"
+    "url": "git://github.com/rsnara/reddit-client.git"
   },
   "bugs": {
-    "url": "https://github.com/rangle/rangle-starter/issues"
+    "url": "https://github.com/rsnara/reddit-client/issues"
```

Comment:
Sigh... `rangle-starter` actually asks for the github username. I wonder why they don't modify the package.json file appropriately. Maybe we should do a pull request.